index.js

```diff
@@ -37,20 +38,18 @@ var sourcemapReporter = function(file) {
       colNum = message.col;
     }
 
-    gutil.log(
-      c.red('[') +
-      (
-        typeof message.line !== 'undefined' ?
-          c.yellow( 'L' + lineNum ) +
-          c.red(':') +
-          c.yellow( 'C' + colNum )
-        :
-          c.yellow('GENERAL')
-      ) +
-      c.red('] ') +
-      message.message + ' ' + message.rule.desc + ' (' + message.rule.id + ')');
+    var msgInfo = message.message + ' ' + message.rule.desc + ' (' + message.rule.id + ')';
+
+    var locInfo = c.red('(') + c.yellow(path.relative(process.cwd(), originalPos.source) + ':');
+    if (typeof message.line !== 'undefined') {
+      locInfo += c.yellow(lineNum) + c.red(':') + c.yellow(colNum);
+    } else {
+      locInfo += c.yellow('GENERAL');
+    }
+    locInfo += c.red(')');
+
+    console.log(msgInfo, locInfo, '\n');
```

Comment:
I'm sorry @fsenart, I missed a few issues.

I think using `gutil.log` is better, but it is `console.log`.
What kind of reasons are you using `console.log`?